error_middleware.go

```diff
@@ -67,8 +66,14 @@ func writeError(log logrus.FieldLogger, w negroni.ResponseWriter, err error) {
 		w.WriteHeader(500)
 	}
 
+	// We log at error level for all 5xx errors as it means there has been an internal service error. With this logging level, we send a Rollbar error.
+	// In all other cases, we log at info level. The status code is most probably a 4xx (i.e. due to a user issue). We don't want a Rollbar error in this case but still want to be informed in the logs.
 	if w.Status()/100 == 5 {
 		log.Error("Request error")
+	} else if isCauseValidationErrors {
+		log.Info("Request validation error")
```

Comment:
I wanted to keep the current behaviour but we can remove this condition if you prefer